rest-api-spec/src/main/resources/rest-api-spec/api/_internal.delete_desired_balance.json

```diff
@@ -2,7 +2,7 @@
   "_internal.delete_desired_balance":{
     "documentation":{
       "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/delete-desired-balance.html",
```

Comment:
>      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/delete-desired-balance.html",

Can we remove these fields for the "internal" APIs? The urls in these cases are incorrect since there's nothing public for them.